crates/humanode-runtime/src/tests/claims_and_vesting.rs

```diff
@@ -176,6 +182,19 @@ fn new_test_ext() -> sp_io::TestExternalities {
             ],
             total_claimable: Some(2 * VESTING_BALANCE),
         },
+        evm: EVMConfig {
+            accounts: {
+                let mut map = BTreeMap::new();
+                let init_genesis_account = fp_evm::GenesisAccount {
+                    balance: INIT_BALANCE.into(),
+                    code: Default::default(),
+                    nonce: Default::default(),
+                    storage: Default::default(),
+                };
+                map.insert(EvmBalancesPot::account_id(), init_genesis_account);
+                map
```

Comment:
Let's use an array/vec of slices form here instead. That'd be a bit cleaner imo.